trunk/SUAVE/Methods/Weights/Correlations/FLOPS/wing_weight.py

```diff
@@ -0,0 +1,487 @@
+## @ingroup Methods-Weights-Correlations-FLOPS
+# wing_weight.py
```

Comment:
Looking at a few files, should this file be named wing.py. Because it already sits in "weights/correlations/flops" . Given than you have Flops/payload and systems instead of Flops/Payload_weigth, consider the name change